tests/lib/rook/upstream.py

```diff
@@ -69,9 +65,11 @@ def build(self):
                 % os.path.join(self.build_dir, 'rook-ceph.tar.gz'))
         self._rook_built = True
 
-    def upload_rook_image(self):
```

Comment:
Why this move of the function? Makes reviews harder that it could be

Reply:
That's because I discovered this bug when testing helm, and cherry picked from other branch where other change were done.
I agree this is not optimal but I wanted to split this from helm PR.